README.md

```diff
@@ -63,13 +63,15 @@ add functionality that cannot easily be added to the Git backend.
 
 ### The working copy is automatically committed
 
-Almost all Jujutsu commands automatically commit the working copy. That means
-that commands never fail because the working copy is dirty (no "error: Your
-local changes to the following files..."), and there is no need for `git stash`.
-You also get an automatic backup of the working copy whenever you run a command.
-Also, because the working copy is a commit, commands work the same way on the
-working-copy commit as on any other commit, so you can set the commit message
-before you're done with the changes.
+Jujutsu uses a real commit to represent the working copy. Checking out a commit
+results a new working-copy commit on top of the target commit. Almost all
+commands automatically amend the working-copy commit. The working-copy being a
```

Comment:
Optional: Perhaps we should split this into two paragraphs, with a paragraph break after "...automatically amend the working-copy commit".

Even more optionally:

We could change the second paragraph into a list:

```
The working-copy being a commit means:

- Commands never fail...
- There is no need for `git stash`
- You get an automatic backup...
- Commands work the same...
```

If that doesn't look good, I'd suggest getting rid of one of these points (either the `git stash` one or the automatic backup one).

Reply:
Makes sense, thanks. I added the paragraph break and dropped the sentence about backup.